src/test/java/StepDefinitions.java

```diff
@@ -0,0 +1,306 @@
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class StepDefinitions {
```

Comment:
Definitions look good to me, short and concise. Could expand on them for future tests.